app/models/hxl_stats_view.rb

```diff
@@ -0,0 +1,109 @@
+class HxlStatsView < ActiveRecord::Base
+  # View aggregates data in a multi-nested-group by for the HXL proxy/endpoint
+  #
+  self.table_name = 'hxl_stats_view'
+  def readonly?
+    true
+  end
+
+  HXLSTATS_COLUMN_HEADERS =
+    ["Emotional State",
+     "Stage of journey",
+     "Country drawn in",
+     "Total children affected",
+     "Children who identify as female",
+     "Children who identify as male",
+     "Children who identify as neither female nor male",
+     "Children between the ages of 5-12",
+     "Children between the ages of 13-18",
+     "Children under 5 years old",
+     "Older than 18 years old"].freeze
+
+  HXL_STATS_TAGS =
+    ["#impact+indicator+code",
+     "#affected+children",
+     "#country+code",
+     "#affected+children+total",
+     "#affected+children+female",
+     "#affected+children+male",
+     "#affected+children+other_gender",
+     "#affected+children+age_5_12",
+     "#affected+children+age_13_18",
+     "#affected+children+age_under5",
+     "#affected+children+age_18plus"].freeze
+
+  private_class_method :get_counts_by_gender
```

Comment:
Ah the 'private class method' debacle! Yeah, wish this was a bit less painful in Rails.

We can use the `private_class_method` option, however this needs to be declared after the methods themselves or Rails complains. A second, think less PITA option, is to use: `class << self` around all of the methods. It's usually clearer to use the `def self.method_name` style to define class methods, however when all of the methods within a class are public, wrapping everything in `class << self` is fine and more readable. Also, this way you can use the `private` keyword as normal.

I.e.

```
class << self
  def results_by_emotional_state
    ...
  end

  private

  def hxl_stats_counts
    ...
  end

  def get_counts_by_gender(hxlstatsmajorgroup)
    ...
  end

  def get_counts_by_age(hxlstatsmajorgroup)
    ...
  end
end
```